modules/standard/GPU.chpl

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2020-2022 Hewlett Packard Enterprise Development LP
+ * Copyright 2004-2019 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* Supports utility functions for operating with GPUs.
+
+  .. warning::
+
+    This module is unstable and its interface is subject to change in the
+    future.
+
+*/
+module GPU
```

Comment:
Should this module have a `--warn-unstable` warning?

Reply:
Maybe.  I was just following the precedent set by `GPUDiagnostics`.  Anything GPU support related is sort of implicitly unstable, but I suppose someone using GPU support may still be interested in finding out what outside of GPU support is unstable.

Anyway assuming these warnings only show up once and that `--warnUnstable` is off by default then I don't see any harm in warning.

I might pose this question to the Chapel 2.0 subteam on the off chance I'm missing something.  In the meantime I'll leave it off this PR and push it onto the list of my smaller issues here: https://github.com/Cray/chapel-private/issues/3692